wifiphisher/data/phishing-pages/wifi_connect/config.ini

```diff
@@ -2,3 +2,7 @@
 Name: Network Manager Connect
 Description: The idea is to imitate the behavior of the network manager by first showing the browser's "Connection Failed" page and then displaying the victim's network manager window through the page asking for the pre-shared key.
 [context]
+# The variable enterprise is used when we are working with Windows enterprise network
+# enterprise = 1 : we show the form for login of enterprise networks to the victim
+# enterprise = 0 : we show the form for login of the non-enterprise networks to the victim 
+enterprise: 1
```

Comment:
The default should remain "0" for now.

Reply:
1) I changed the enterprise value back to 0.

2) Also, I made changes to fix issues like in the image above in both MacOS and Windows.